content/millennium/r4/encounters/encounter.md

```diff
@@ -97,3 +97,56 @@ The common [errors] and [OperationOutcomes] may be returned.
 [`_count`]: http://hl7.org/fhir/R4/search.html#count
 [errors]: ../../#client-errors
 [OperationOutcomes]: ../../#operation-outcomes
+
+## Create
+
+Create an individual Encounter.
+
+    POST /Encounter
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/json+fhir'} %>
+
+### Body Fields
+
+<%= definition_table(:encounter, :create, :r4) %>
```

Comment:
this table scrolls... any way we can only scroll the code sections instead of the whole thing

Reply:
I'm not sure if we have any fine control over this? Patient works this way too: https://fhir.cerner.com/millennium/dstu2/individuals/patient/#body-fields